components/scream/src/share/io/tests/io.cpp

```diff
@@ -76,6 +76,20 @@ TEST_CASE("input_output_basic","io")
     m_output_manager.run(time);
   }
   m_output_manager.finalize();
+  // Because the next test for input in this sequence relies on the same fields.  We set all field values
+  // to nan to ensure no false-positive tests if a field is simply not read in as input.
+  for (auto it : out_fields)
+  {
+    auto f_dev  = field_repo->get_field(it,"Physics").get_view();
+    auto f_host = Kokkos::create_mirror_view( f_dev );
+    Kokkos::deep_copy(f_host,f_dev);
```

Comment:
You could simply do
```
Kokkos::deep_copy(f_dev,std::nan(""));
```
The `deep_copy` function is overloaded, and can either copy a src view into a dst view or a single value into a dst view.